doc/interaction/README.md

```diff
@@ -47,8 +47,6 @@ awslocal kinesis list-streams
 }
 ```
 
-**UPDATE**: Use the environment variable `$LOCALSTACK_HOSTNAME` to determine the target host inside your Lambda function. See [Configuration](#configuration) section for more details.
```

Comment:
@HarshCasper Are these outdated READMEs under [docs](https://github.com/localstack/localstack/tree/master/doc) still relevant here. Maybe we should remove them and link to our documentation page instead?